criu/include/cr_options.h

```diff
@@ -70,7 +70,15 @@ enum NETWORK_LOCK_METHOD {
 	NETWORK_LOCK_SKIP,
 };
 
+/**
+ * CRIU currently defaults to the iptables locking backend.
+ *
+ * It is, however, possible to change this by defining
+ * NETWORK_LOCK_DEFAULT to a different value on the command-line.
+ */
+#ifndef NETWORK_LOCK_DEFAULT
 #define NETWORK_LOCK_DEFAULT NETWORK_LOCK_IPTABLES
```

Comment:
should we change the default to nftable?

Reply:
I don't know. Currently Fedora >= 42 defaults to nft and CentOS Stream >= 10 (RHEL >= 10).

@danishprakash mentioned that Suse is also looking into switching to nft. Not sure what other distributions are planing. 

Maintaining the RHEL and Fedora packages it would make my life a bit easier, but I still need to deal with older Fedora and RHEL releases. It feels a bit too early. Maybe if Debian or Ubuntu also switches to nft, the we would have most of the bigger distributions.